scripts/jira_oauth_setup_script.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2020 Google, LLC.
```

Comment:
A general comment, that can either be addressed in this PR or a separate PR: I think we need to start working on the README. This seems like a setup step that should be documented there.

Reply:
For sure, I just created an issue to update the readme and specified that instructions to run this script should be added.